File: model/src/main/java/org/pulcifer/climate/model/ClimateMonth.java
package org.pulcifer.climate.model;

import com.fasterxml.jackson.annotation.JsonInclude;

import java.math.BigDecimal;

@JsonInclude(JsonInclude.Include.NON_NULL)
public class ClimateMonth {
    Integer month;
    BigDecimal maxTemp;
    BigDecimal minTemp;
    BigDecimal meanTemp;
    BigDecimal maxTempF;
    BigDecimal minTempF;
    BigDecimal meanTempF;
    BigDecimal raindays;
    BigDecimal rainfall;

    public ClimateMonth() {}

    public Integer getMonth() {return this.month;}

    public BigDecimal getMaxTemp() {return this.maxTemp;}

    public BigDecimal getMinTemp() {return this.minTemp;}

    public BigDecimal getMeanTemp() {return this.meanTemp;}

    public BigDecimal getMaxTempF() {return this.maxTempF;}

    public BigDecimal getMinTempF() {return this.minTempF;}

    public BigDecimal getMeanTempF() {return this.meanTempF;}

    public BigDecimal getRaindays() {return this.raindays;}

    public BigDecimal getRainfall() {return this.rainfall;}

}

File: src/main/kotlin/org/pulcifer/climate/model/ClimateMonth.kt
package org.pulcifer.climate.model

import com.fasterxml.jackson.annotation.JsonInclude
import java.math.BigDecimal

@JsonInclude(JsonInclude.Include.NON_NULL)
class ClimateMonth {
    var month: Int? = null
    var maxTemp: BigDecimal? = null
    var minTemp: BigDecimal? = null
    var meanTemp: BigDecimal? = null
    var maxTempF: BigDecimal? = null
    var minTempF: BigDecimal? = null
    var meanTempF: BigDecimal? = null
    var raindays: BigDecimal? = null
    var rainfall: BigDecimal? = null
}